code/src/Consumption.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+ 
+const consumptionOptions = [
+    "read the printed book",
+    "read the e-book",
+    "listen to the audio book"
+]
+
+export const Consumption = ({ favConsumption, onConsumptionChange }) => {
+    return (
+        <>
+            <section className="question">
+                <label htmlFor="consumption" className="dropdown">
+                    How do you prefer to enjoy a book?
```

Comment:
not sure, but i could not use the keyboard to tab the text " how do you prefer to enjoy a book?". The screen reader reads it but i think it should be accessible even when only using keyboard? :)

Reply:
Added a tabindex and now it works